packages/gasket-plugin-redux/test/index.spec.js

```diff
@@ -73,15 +74,56 @@ describe('Plugin', () => {
     });
   });
 
-  describe('webpack', () => {
+  describe('webpackConfig', () => {
     it('has expected hook', () => {
-      expect(plugin.hooks).toHaveProperty('webpack', expect.any(Function));
+      expect(plugin.hooks).toHaveProperty('webpackConfig', expect.any(Function));
     });
 
     it('adds GASKET_MAKE_STORE_FILE to EnvironmentPlugin', function () {
       mockGasket.config.redux = mockReduxConfig;
 
-      results = plugin.hooks.webpack(mockGasket, {});
+      results = plugin.hooks.webpackConfig(mockGasket, {}, { webpack });
+      expect(results).toHaveProperty('plugins', expect.arrayContaining([
+        expect.objectContaining({
+          defaultValues: {
+            GASKET_MAKE_STORE_FILE: mockReduxConfig.makeStore
+          }
+        })
+      ]));
+    });
+
+    it('merges with existing plugins', function () {
+      mockGasket.config.redux = mockReduxConfig;
+      const mockWebpackConfig = {
+        plugins: [
+          new webpack.EntryOptionPlugin(),
+          new webpack.DynamicEntryPlugin()
+        ]
+      };
+
+      results = plugin.hooks.webpackConfig(mockGasket, mockWebpackConfig, { webpack });
+      expect(results.plugins).toHaveLength(3);
+      expect(results).toHaveProperty('plugins', expect.arrayContaining([
+        expect.objectContaining({
+          defaultValues: {
+            GASKET_MAKE_STORE_FILE: mockReduxConfig.makeStore
+          }
+        })
+      ]));
+    });
+
+    it('merges with existing EnvironmentPlugin', function () {
+      mockGasket.config.redux = mockReduxConfig;
+      const mockWebpackConfig = {
+        plugins: [
+          new webpack.EnvironmentPlugin([
```

Comment:
If there are two instances of `EnvironmentPlugin` do they end up merging into what `process.env` contains? Just want to make sure this doesn't cause any new overwriting behavior.

Reply:
👍🏻 Yes that's correct